mappings/net/minecraft/client/util/profiler/SamplingRecorder.mapping

```diff
@@ -0,0 +1,59 @@
+CLASS net/minecraft/class_5965 net/minecraft/client/util/profiler/SamplingRecorder
+	FIELD field_29598 metric Lnet/minecraft/class_5951;
+	FIELD field_29599 timeGetter Ljava/util/function/DoubleSupplier;
+	FIELD field_29600 buffer Lio/netty/buffer/ByteBuf;
+	FIELD field_29601 isActive Z
+	FIELD field_29602 startAction Ljava/lang/Runnable;
+	FIELD field_29603 writeAction Lnet/minecraft/class_5965$class_5967;
+	METHOD <init> (Lnet/minecraft/class_5951;Ljava/util/function/DoubleSupplier;Ljava/lang/Runnable;Lnet/minecraft/class_5965$class_5967;)V
+		ARG 1 metric
+		ARG 2 timeGetter
+		ARG 3 startAction
+		ARG 4 writeAction
+	METHOD method_34775 length ()I
```

Comment:
Is this the length of the backing buffer?

Reply:
Yes.